src/EPR.SubsidiaryBulkUpload.Application/Services/BulkSubsidiaryProcessor.cs

```diff
@@ -21,7 +21,7 @@ public async Task Process(IEnumerable<CompaniesHouseCompany> subsidiaries, Compa
             .SelectAwait(async subsidiary => (Subsidiary: subsidiary, SubsidiaryOrg: await organisationService.GetCompanyByCompaniesHouseNumber(subsidiary.companies_house_number)));
 
         var subsidiariesAndOrgWithValidName = subsidiariesAndOrg
-            .Where(sub => sub.SubsidiaryOrg != null && sub.Subsidiary.companies_house_number == sub.SubsidiaryOrg.companiesHouseNumber && sub.Subsidiary.organisation_name == sub.SubsidiaryOrg.name);
+            .Where(sub => sub.SubsidiaryOrg != null && sub.Subsidiary.companies_house_number == sub.SubsidiaryOrg.companiesHouseNumber && sub.Subsidiary.organisation_name.ToLower() == sub.SubsidiaryOrg.name.ToLower());
```

Comment:
String comparisons should use string.Compare with case insensitive comparison options. It's faster and less likely to end up in an exception if the string is unexpectedly null. There are other string comparisons in the lines below.

Reply:
changed